### PR TITLE
Abdulbawab

### PR DESCRIPTION
**Pull request by Albawab**

### CODE REVIEW

File: js1-2-1.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+let Arabic = "مرحبا ايها العالم",
```
Comment (marcianoviereck) [inline js1-2-1.js:1]: Fancy! This is also a nice way to do this!

File: js1-2-10.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+let  multipleTypes = [2, "Sam", true];
```
Comment (marcianoviereck) [inline js1-2-10.js:1]: This can be `const` since the value won't change anymore

File: js1-2-3.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+console.log("the value of my variable x will be: undefined");
+
+// the value of x
+console.log(typeof(x));
+
+// add an value 
+  x = 5;
+
+  // the value of x
+  console.log("the value of x will be: number");
```
Comment (marcianoviereck) [inline js1-2-3.js]: I don't think the value will be `number`:)

File: js1-2-5.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+ let z = 7.25;
+ console.log(z);
+
+ let a = Math.round(z);
+ console.log(a);
+
+ // compare the two values
+ //Math.max(z,a);
+
+ console.log(Math.max(z,a));
```
Comment (marcianoviereck) [inline js1-2-5.js:14]: Ah nice! Good solution!

File: js1-2-6.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+// Declare an empty array
+let moreAnimals = [];
+let more_Sports = new Array();
```
Comment (marcianoviereck) [inline js1-2-6.js]: You can remove this one, since you don't use it

File: js1-2-8.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+let firstNumber = 7,
+    secondNumber = 10;
+
+console.log("The value of my variable foo is: " + firstNumber);
```
Comment (marcianoviereck) [inline js1-2-8.js]: foo = firstNumber

it's fine, just that you wrote `foo` instead of `firstNumber`:)

File: js1-2-8.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+let firstNumber = 7,
+    secondNumber = 10;
```
Comment (marcianoviereck) [inline js1-2-8.js]: I think you can remove`secondNumber` because it is not used

File: js1-2-8.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+console.log("The value of my variable foo is: " + firstNumber);
+console.log(typeof(firstNumber));
+
+if(typeof(firstNumber) == typeof(secondNumber)){
+    console.log("SAME TYPE");
+}
+
+let firstString = "10";
+
+if(typeof(firstNumber) == typeof(firstString)){
```
Comment (marcianoviereck) [inline js1-2-8.js]: I think you forgot to add more variable (with different types):

```
8.1 First declare at least four variables and assign them different data types.
```

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
The homework looks good! and nice!

I think you forgot a small part of the homework though

File: js1-2-8.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    console.log("SAME TYPE");
+}
+
+
+/* let firstNumber = 7,
+secondNumber = 10; */
+
+let firstString = "10",
+    first = 0;
+
```
Comment (marcianoviereck) [inline js1-2-8.js:17]: Ah, I think it's almost correct, I think you need to add two variables:

one for the `array` data-type and one of the `boolean` data type. Then compare those as well. Then the homework is approved!

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
One last thing, I think you forgot two data-types in the `js-1-2-8.js` file

**Comment by wouterkleijn** [comment]
Thanks @marcianoviereck for providing feedback. @Albawab Can you please address Marciano's comments? Thanks!

File: js1-2-8.js [hunk 10]
```diff
@@ -12,8 +12,8 @@ if(typeof(firstNumber) == typeof(secondNumber)){
 /* let firstNumber = 7,
 secondNumber = 10; */
 
-let firstString = "10",
-    first = 0;
+let firstString = ture,
```
Comment (marcianoviereck) [inline js1-2-8.js:15]: this is not valid a boolean value (and will not compile). If you can fix this then it is fine

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
I found one last thing that is not correct yet.

**Pull request closed without merging**